starter-code/clue.js

```diff
@@ -1,103 +1,139 @@
-  age: 36
-  image: https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-  occupation: Socialité
-
- Mr Mustard
-  first name: Jack
-  last name: Mustard
-  color: yellow
-  description: He is a former football player who tries to get by on his former glory
-  age: 62
-  image: https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-  occupation: Retired Football player
-
----- Weapons ----
-
-Rope - Weight: 10
-Knife - Weight: 8
-Candlestick - Weight: 2
-Dumbbell - Weight: 30
-Poison - Weight: 2
-Axe - Weight: 15
-Bat - Weight: 13
-Trophy - Weight: 25
-Pistol - Weight: 20
-
-
----- Rooms ----
-Dinning Room
-Conservatory
-Kitchen
-Study
-Library
-Billiard Room
-Lounge
-Ballroom
-Hall
-Spa
-Living Room
-Observatory
-Theater
-Guest House
-Patio
-
-*/
+  image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+  occupation: "Actor",
+};
+
+var mrsPeacock = {
+  name: "Eleanor Peacock",
+  colour: "blue",
+  description: "She is from a wealthy family and uses her status and money to earn popularity",
+  age: 36,
+  image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+  occupation: "Socialité",
+};
+
+var mrMustard = {
+  name: "Jack Mustard",
+  colour: "yellow",
+  description: "He is a former football player who tries to get by on his former glory",
+  age: 62,
+  image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+  occupation: "Retired Football player",
+};
+
+
+var rope = {
+  name: "Rope",
+  weight: 10,
+};
+
+var knife = {
+  name: "Knife",
+  weight: 8,
+};
+
+var candlestick = {
+  name: "Candlestick",
+  weight: 2,
+};
+
+var dumbell = {
+  name: "Dumbell",
+  weight: 30,
+};
+
+var poison = {
+  name: "Poison",
+  weight: 2,
+};
+
+var axe = {
+  name: "Axe",
+  weight: 15,
+};
+
+var bat = {
+  name: "Bat",
+  weight: 13,
+};
+
+var trophy = {
+  name: "Trophy",
+  weight: 25,
+};
+
+var pistol = {
+  name: "Pistol",
+  weight: 20,
+};
+
+
+var rooms = [{name: "Dining Room",},
+             {name: "Conservatory",},
+             {name: "Kitchen",},
+             {name: "Study",},
+             {name: "Library",},
+             {name: "Billiard Room",},
+             {name: "Lounge",},
+             {name: "Ballroom",},
+             {name: "Hall",},
+             {name: "Spa",},
+             {name: "Living Room",},
+             {name: "Observatory",},
+             {name: "Theatre",},
+             {name: "Guesthouse",},
+             {name: "Patio"},
+             ];
+
+var suspects = [];
+suspects.push(mrMustard, mrsPeacock, missScarlet, profPlum, drOrchid, mrGreen);
+
+var weapons =[];
+weapons.push(pistol,trophy,bat,axe,poison,dumbell,candlestick,knife,rope);
 
 function random_selector(array) {
+    var randomNumber = Math.floor(Math.random()*array.length);
+    return array[randomNumber].name;
 
 }
 
 function pick_mistery() {
-
+    return caseFile = {suspect: random_selector(suspects), weapon: random_selector(weapons), room: random_selector(rooms)};
```

Comment:
This works but it doesn't do what you think it does. I suggest that you do instead:

```
return {
  suspect: random_selector(suspects),
  weapon: random_selector(weapons),
  room: random_selector(rooms)
};
```

By the way, I like that you named the property `suspect` instead of `character`. It has a much more precise meaning.